### ISSUE TITLE
Issue while Installing

### ISSUE THREAD

**Issue by joshluongo** [lvillani/carthage-copy-frameworks#11]
When I tried to install this on a fresh install I got this error from Brew

```
==> Installing carthage-copy-frameworks from lvillani/tap
==> Downloading https://github.com/lvillani/carthage-copy-frameworks/archive/v1.2.0.tar.gz
Already downloaded: /Users/build/Library/Caches/Homebrew/downloads/d6d8389a5c7d79e56ff3b4ca996046b302ce8bfa15e4243b129f9061bd4b4344--carthage-copy-frameworks-1.2.0.tar.gz
==> Downloading https://github.com/pypa/virtualenv/archive/16.2.0.tar.gz
Already downloaded: /Users/build/Library/Caches/Homebrew/downloads/bfb26e5e9fbf90082f14312f9f0af39c489fc665de9adcc2c003f3e4c533bfd6--virtualenv-16.2.0.tar.gz
==> python2.7 -c import setuptools... --no-user-cfg install --prefix=/private/tmp/carthage-copy-frameworks--homebrew-virtualenv-20190116-24151-c32up7/target --install-scripts=/private/tmp/carthage-copy-frameworks--homebrew-virtualenv-20190116-24151-c32up7/target/bin --sin
==> python2.7 -s /private/tmp/carthage-copy-frameworks--homebrew-virtualenv-20190116-24151-c32up7/target/bin/virtualenv -p python2.7 /usr/local/Cellar/carthage-copy-frameworks/1.2.0/libexec
Last 15 lines from /Users/build/Library/Logs/Homebrew/carthage-copy-frameworks/02.python2.7:
2019-01-16 16:40:47 +1100

python2.7
-s
/private/tmp/carthage-copy-frameworks--homebrew-virtualenv-20190116-24151-c32up7/target/bin/virtualenv
-p
python2.7
/usr/local/Cellar/carthage-copy-frameworks/1.2.0/libexec

/System/Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python: can't open file '/private/tmp/carthage-copy-frameworks--homebrew-virtualenv-20190116-24151-c32up7/target/bin/virtualenv': [Errno 2] No such file or directory

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/lvillani/homebrew-tap/issues
```

**Comment by joshluongo** [comment]
Sorry I just solved it by install Python 2.7 from Brew.

```bash
brew install python@2
```

**Comment by lvillani** [comment]
Hi @joshluongo! You found a bug in my Homebrew formula! I opened issue #12 to track it down so that it doesn't happen in the future.
